src/pages/[id].page.jsx

```diff
@@ -54,52 +62,54 @@ export default function Details({ details }) {
         <RichText markdown={definition} />
       </Definition>
       <div className="details-page__primary">
-        <article className="sandpack">
-          <header>
-            <h2>Code Example</h2>
-          </header>
-          <figure>
-            <Sandpack
-              template="vanilla"
-              theme={levelUp}
-              files={{
-                '/index.html': {
-                  code: htmlCode,
-                  active: true,
-                },
-                'index.css': cssCode,
-                '/index.js': javascriptCode,
-                '/src/index.js': {
-                  code: 'import "../index.css"; import "../index.js"',
-                  hidden: true,
-                },
-              }}
-              options={{
-                showLineNumbers: true,
-                wrapContent: true,
-                editorHeight: 480,
-              }}
-            />
-          </figure>
-          <figcaption>
-            Once focused on the code editor:
-            <p>
-              Press
-              {' '}
-              <kbd>Enter</kbd>
-              {' '}
-              to enter the code editor and change the
-              code.
-            </p>
-            <p>
-              Press
-              {' '}
-              <kbd>Escape</kbd>
-              {' '}
-              to leave the code editor.
-            </p>
-          </figcaption>
-        </article>
+        {canRenderSandpack ? (
```

Comment:
I didn't realize the `<article>` was the only child of this `div`. We can move the conditional rendering logic up a level, covering `<div className="details-page__primary">`.

Reply:
If I go up to the `details-page__primary` div, I get this!! 
<img width="1434" alt="Screen Shot 2022-12-14 at 12 23 41 PM" src="https://user-images.githubusercontent.com/69602589/207664486-61028386-264b-4b52-a565-8af739f0f231.png">

Reply:
I see what you mean though, that the div is still showing up...
<img width="1388" alt="Screen Shot 2022-12-14 at 12 25 52 PM" src="https://user-images.githubusercontent.com/69602589/207665061-461f2110-cbee-4e71-a527-bef7c74f938b.png">

Reply:
Oh weird, do we have negative margins or some special positioning logic on the Usage Guidelines? If so, we might want to change that. If it's a huge hassle, an empty `div` isn't the worst thing.

Reply:
Yes- I think it was to prevent a bunch of extra blank, black space between the definition and the old code components.

It's still a little blank looking to me, but we can get rid of the negative margin. Here are screen shots without the negative margin, and including the div in the conditional render. 

<img width="1386" alt="Screen Shot 2022-12-14 at 1 04 37 PM" src="https://user-images.githubusercontent.com/69602589/207672801-c062d551-85a1-4a79-a5aa-bb5dac4f1763.png">
<img width="1348" alt="Screen Shot 2022-12-14 at 1 04 54 PM" src="https://user-images.githubusercontent.com/69602589/207672802-0b9243fb-272a-409b-8db3-3cb8513a2347.png">

Reply:
This is the commit with these changes: https://github.com/sparkbox/accessible-components/pull/132/commits/7915cc4a5d5c111de5af61b97a5bdff303980d8a